common/app/implicits/Requests.scala

```diff
@@ -30,7 +30,9 @@ trait Requests {
 
     lazy val isAsset = r.path.contains("/assets/")
 
-    lazy val isHtml = !(isAsset || isJson || isRss || isEmail || isXmlHttpRequest || isAmp || isHealthcheck)
+    lazy val isAlert = r.path.contains("/news-alert/")
+
+    lazy val isHtml = !(isAlert || isAsset || isJson || isRss || isEmail || isXmlHttpRequest || isAmp || isHealthcheck)
```

Comment:
should this `isHtml` look into the `content-type` rather than excluding endpoints based on their url?